HorizonNing/src/main/java/com/hubspot/horizon/ning/NingAsyncHttpClient.java

```diff
@@ -137,7 +141,7 @@ private ListenableFuture<HttpResponse> internalExecute(
       retryHandler,
       mapper
     );
-    final Request ningRequest = requestConverter.convert(request);
+    final Request ningRequest = requestConverter.convert(request, customDnsResolver);
```

Comment:
Does this have to be set per-request or is there a way to set it globally on the client?

Reply:
Unfortunately no, I could only find a way to set it [per-request](https://github.com/AsyncHttpClient/async-http-client/blob/fb50dc26717f0e6aaaef58e2a01924a56aab2021/client/src/main/java/org/asynchttpclient/RequestBuilderBase.java#L570)